wallet-core/wallet-operations.md

```diff
@@ -90,10 +98,83 @@ This can be done in two steps: first a derived private key is obtained, then an
 > Avoid any risk of leakage of secrets!
 
 ```swift
-let key = wallet.getKey(derivationPath: "m/44'/60’/1’/0/0")
+let key = wallet.getKey(derivationPath: "m/44\'/60\'/1\'/0/0")   // m/44'/60'/1'/0/0
 let address = CoinType.ethereum.deriveAddress(privateKey: key)
 ```
 
 For example, a second Ethereum address can be derived using the custom derivation path `”m/44'/60’/1’/0/0”` (note the 1 in the third position),
 yielding address `0x68eF4e5660620976a5968c7d7925753D3Cc40809`.
 
+
+## Transaction Signing
+
+In general, when creating a new blockchain transaction, a wallet has to:
+
+1. Put together a transaction with relevant fields (source, target, amount, etc.)
+2. Sign the transaction, using the account private key.  This is done by Wallet Core.
+3. Send to a node for broadcasting to the blockchain network.
+
+The exact fields needed for a transaction are different for each blockchain.
+In Wallet Core, signing input and output parameters are typically represented in a protobuf message
+(internally needed for serialization for passing through different language runtimes).
+
+A generic, coin-independent signer also exists (*AnySigner*), but its usage is recommended only in browser-based applications.
+
+### Ethereum Transaction Signing
+
+A simple Ethereum send transaction needs the following fields:
+
+Field | Sample value | Description
+---|---|---
+chainID | 1 | Network selector, use 1 for mainnet
```

Comment:
You can mention source for other chains https://chainid.network/

Reply:
👍